code/modules/religion/honorbound/honorbound_trauma.dm

```diff
@@ -95,14 +105,15 @@
  */
 /datum/brain_trauma/special/honorbound/proc/is_honorable(mob/living/carbon/human/honorbound_human, mob/living/target_creature)
 	var/is_guilty = (target_creature in guilty)
+	var/is_human = ishuman(target_creature)
 	//THE UNREADY (Applies over ANYTHING else!)
 	if(honorbound_human == target_creature)
 		return TRUE //oh come on now
 	if(target_creature.IsSleeping() || target_creature.IsUnconscious() || HAS_TRAIT(target_creature, TRAIT_RESTRAINED))
 		to_chat(honorbound_human, span_warning("There is no honor in attacking the <b>unready</b>."))
 		return FALSE
 	//THE JUST (Applies over guilt except for med, so you best be careful!)
-	if(ishuman(target_creature))
+	if(is_human)
```

Comment:
This could be an early return. Also, borgs are considered lesser creatures by this standard.

Reply:
Borgs being lesser creatures makes sense as far as the sect flavor goes, but do you think that would be an issue in terms of gameplay? Maybe it's OK for honorbound to attack borgs first... eh, I'll change it.

Reply:
I'm ambivalent to it to be fair.

Reply:
I'm down